app/controllers/ansible_roles_controller.rb

```diff
@@ -0,0 +1,42 @@
+#:nodoc:
+class AnsibleRolesController < ::ApplicationController
```

Comment:
Nitpick - put this in the `ForemanAnsible` namespace, then just call it RolesController. It'll allow us to not override stuff in ApplicationController like on line 19 (`process_error :object => @role`) would just become `process_error`